camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -637,11 +637,14 @@ def cal_fromcorners(self, good):
         # If FIX_ASPECT_RATIO flag set, enforce focal lengths have 1/1 ratio
         self.intrinsics[0,0] = 1.0
         self.intrinsics[1,1] = 1.0
-        cv2.calibrateCamera(
+        reproj_err, self.intrinsics, self.distortion, rvecs, tvecs = cv2.calibrateCamera(
```

Comment:
I’m slightly unclear why we expect the distortion params on return to be different from those added to the self.distortion implicitly. Is this the result of a bug in opencv? If this is true the API reads to me that those are optional parameters you can simply remove and just use the direct return copies.

Reply:
As I understand it, the Python API of OpenCV does not intend the function arguments that are used for output in C++ to be used as such in Python and the return values should be used in their place instead. Considering this, I think the fact that the output variables returned something reasonable earlier was just a lucky coincidence.

Passing all-zero `self.distortion` as input is redundant, indeed, as it matches the default behavior. I'll remove it from the arguments.